### PR TITLE
lint[tests]: VariableNamingDetector

### PR DESCRIPTION
**Pull request by Arnold2381**
## Fixes
Related: #9538 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/AnalyticsTest.java [hunk 1]
```diff
@@ -119,7 +119,7 @@ public void testSendException() {
                     .thenReturn(new SpyGoogleAnalyticsBuilder());
 
             // This is actually a Mockito Spy of GoogleAnalyticsImpl
-            GoogleAnalytics mAnalytics = UsageAnalytics.initialize(mMockContext);
+            GoogleAnalytics Analytics = UsageAnalytics.initialize(mMockContext);
```
Comment (david-allison-1) [inline AnkiDroid/src/test/java/com/ichi2/anki/AnalyticsTest.java:122]:
```suggestion
            GoogleAnalytics analytics = UsageAnalytics.initialize(mMockContext);
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Thanks!

Variable name needs to be lowercase. I've proposed one change, but this applies to all

**Review by david-allison-1 — APPROVED** [review]
LGTM, thanks!

**Review by mikehardy — APPROVED** [review]
cool thanks!

**Pull request merged**